rust/sdk-examples/src/utils/starting_version.rs

```diff
@@ -41,6 +44,18 @@ pub async fn get_latest_processed_version_from_db(
 ) -> Result<Option<u64>> {
     let mut conn = conn_pool.get().await?;
 
+    if let Some(backfill_config) = &indexer_processor_config.backfill_config {
+        return match BackfillProcessorStatusQuery::get_by_processor(
```

Comment:
Would this query every take into account of the backfill status?

Reply:
Yep exactly. We would change some logic depending on what the backfill status of the retrieved row is